knx/describe.go

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2022 mobilarte.
+// Licensed under the MIT license which can be found in the LICENSE file.
+
+package knx
+
+import (
+	"time"
+
+	"github.com/vapourismo/knx-go/knx/knxnet"
+)
+
+// Describe a single KNXnet/IP server.
+func Describe(Address string, searchTimeout time.Duration) (*knxnet.DescriptionRes, error) {
+	// Uses a UDP socket.
+	socket, err := knxnet.DialTunnel(Address)
+	if err != nil {
+		return nil, err
+	}
+	defer socket.Close()
+
+	addr, _ := socket.LocalAddr()
+
+	req, err := knxnet.NewDescriptionReq(addr)
```

Comment:
If the description request depends on the socket sending it, due to its local address, maybe just pass in the socket directly?

Reply:
Possible, but I wanted to have the same parameters for `NewDescriptionReq` and `NewSearchReq`.

Reply:
Makes sense.